src/App.js

```diff
@@ -1,8 +1,23 @@
 import React from 'react';
+import Sum from './components/Sum';
 import './App.scss';
 
 const App = () => (
-  <h1>React sum</h1>
+  <section>
+    <h1>React sum</h1>
+    <Sum
+      a={2}
+      b={2}
+    />
```

Comment:
no need to push every prop to the new line if they don't consume too much space horizontally

```suggestion
    <Sum a={2} b={2} />
```